css/onoffice-default.css

```diff
@@ -66,4 +66,7 @@ form[data-applicant-form-id] #spinner {
     display: inline;
     width: 40px;
     vertical-align: middle;
+}
+.oo-listinfo .oo-listinfotable .oo-detailslisttd {
```

Comment:
Could you please move this CSS into the template, because this needs to stay backwards compatible and can barely be changed in the future.

Reply:
> Could you please move this CSS into the template, because this needs to stay backwards compatible and can barely be changed in the future.

@jayay , i fixed it, can you check it help me ?